doc/docs/master/how-tos/create-pipeline.md

```diff
@@ -0,0 +1,95 @@
+# Create a Pipeline
+
+Pipelines are computational entities that run your code. A standard
+pipeline injects data from an input repository into your code, runs
+your code against that data, and places the results into an output
+repository within the Pachyderm file system. To create a pipeline,
+you need to define a pipeline specification in the JSON or YAML file
+format.
+
+This is a simple example of a Pachyderm pipeline specification:
+
+```# edges.json
+{
+  "pipeline": {
+    "name": "edges"
+  },
+  "description": "A pipeline that performs image edge detection by using the OpenCV library.",
+  "transform": {
+    "cmd": [ "python3", "/edges.py" ],
+    "image": "pachyderm/opencv"
+  },
+  "input": {
+    "pfs": {
+      "repo": "images",
+      "glob": "/*"
+    }
+  }
+}
+```
+
+At the very minimum, a standard pipeline needs to have a name, a
+transformation code, a Docker image with your code, and an input
+repository with a branch and glob pattern specified. Special types
```

Comment:
What is `transform`? I understand what it is because I've used it before, but I think it might be a bit confusing to new users. I can say *a user code specified in the transformation section* or something similar. Similarly, saying just `an input with a glob pattern.` is confusing. Saying `an input repository` is less ambiguous.